### PR TITLE
Tag edit shelter review us5

### PR DESCRIPTION
**Pull request by tony-gomes**
User Story 5, Edit/Update Review
- edit test
- link on shelters#show page
- edit/update controllers
- edit/update view

### CODE REVIEW

File: app/views/reviews/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<h2>Edit Review</h2>
+
+<section id="edit-review-#{review.id}">
+  <%= form_tag("/shelters/#{@review.shelter_id}/reviews/#{@review.id}", method: :patch)  do %>
```
Comment (mikez321) [inline app/views/reviews/edit.html.erb]: Nice work already implementing this stuff!

File: app/views/shelters/show.html.erb [hunk 2]
```diff
@@ -11,11 +11,13 @@
   <section class="review-section">
     <%= link_to ("New Review"), "/shelters/#{@shelter.id}/reviews/new" %>
     <% @shelter.reviews.each do |review| %>
-      <%= image_tag ( review.review_image ) %>
-      <%= review.title %>
-      Rating: <%= review.rating %>
-      <%= review.content %>
+      <div id="review-<%= review.id %>">
```
Comment (elguapogordo) [inline app/views/shelters/show.html.erb]: always trying to remember where and when to interpolate a certain way... this gets me a lot

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -10,10 +10,19 @@ def create
     redirect_to "/shelters/#{shelter.id}"
   end
 
+  def edit
+    @review = Review.find(params[:review_id])
+  end
+
+  def update
```
Comment (mikez321) [inline app/controllers/reviews_controller.rb]: Do you need instance variables here?  If this is just for the update thing I don't think you do....

File: db/seeds.rb [hunk 4]
```diff
@@ -5,9 +5,11 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
 Pet.destroy_all
 Shelter.destroy_all
 
+# Shelters
```
Comment (mikez321) [inline db/seeds.rb]: Oh, nice thinking putting some notes in the seeds file

**Pull request merged**